demo/ts/components/victory-axis-demo.tsx

```diff
@@ -269,6 +269,14 @@ export default class VictoryAxisDemo extends React.Component<
               "Mariners\nSEA",
             ]}
           />
+
+          <VictoryAxis
+            label="Empty Values"
+            padding={{ top: 40, bottom: 40, left: 40, right: 90 }}
+            orientation="right"
+            style={styleOverrides}
+            tickValues={[]}
+          />
```

Comment:
We have a storybook for this repo too, which takes screenshots of all stories.  (Storybook is separate from the `demo` folder).
Can you please add this test-case to the Storybook story too?

Reply:
will do